go/web/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@rules_webtesting//web:web.bzl", "web_test_config")
+
+package(
+    default_testonly = True,
+    default_visibility = ["//visibility:public"],
```

Comment:
pls limit the visibility or hide under //web/internal/

Reply:
How is this used by web_testing_go?

>  why limit the visibility?

prevent folks to depend on this package directly instead of using web_testing_go